openedx/core/djangoapps/course_groups/cohorts.py

```diff
@@ -137,7 +137,6 @@ def get_cohorted_commentables(course_key):
     return ans
 
 
-@transaction.commit_on_success
```

Comment:
There are no set of queries that need atomicity so we don't really need to replace this with `atomic` and this function is called on every courseware page request, so to avoid an extra pair of savepoint setup teardown queries, its better to just remove it.